_posts/2016-07-26-foreman-community-newsletter-july-2016.md

```diff
@@ -0,0 +1,132 @@
+---
+layout: post
+title: Foreman Community Newsletter - July 2016
+date: 2016-07-26
+author: Greg Sutcliffe
+tags:
+- foreman
+- newsletter
+excerpt: |
+  Our 7th birthday! Also, Foreman 1.12 released, 1.11.4, Katello 3.1 in RC,
+  Red Hat Summit roundup, plugin releases, ... a busy month!
+---
+
+Time again for the Foreman Newsletter - slightly late perhaps, but definitely
+not forgotten! It's been a busy month, and all the details are coming right up.
+
+### Foreman's 7th Birthday!
+
+On July 13th, Foreman turned 7 years old! Parties have been happening in some
+of our communities (see the [birthday events][birthday] page) and there's still
+a few to come.
+
+[birthday]: https://theforeman.org/2016/06/foremans-7th-birthday-events.html
+
+7 years is quite a milestone - and we're still growing! We'd like to thank
```

Comment:
"we're still growing" [citation needed] :smile: 

Not challenging it, I just don't know if that's true or not!